README.md

```diff
@@ -87,18 +86,21 @@ And now it will hold the value `4`.
 
-A string is a type of object which is used to store and manipulate text. In Objective-C, the base string class is `NSString`. We can create a string variable in much the same way as creating an integer variable.
+A string is a type of object which is used to store and manipulate text. While the string literal can be used in conjunction with `NSLog()` to print directly to the console:
+
+```obj
+NSLog(@"%@", @"Welcome to the Flatiron School!");
+```
+This will print: `Welcome to the Flatiron School!`.
+
+In Objective-C, we can also capture a string value into a variable that is of the `NSString` type. This is because the string literal that we've been using implicitly creates values of the `NSString` type. We can create a string *variable* in much the same way as creating an integer variable, however, since strings are *objects*, their declaration syntax requires a `*` ("star") between the type name and the variable name:
 
 ```objc
 NSString *welcome = @"Welcome to the Flatiron School!";
 ```
-You will notice, however, that unlike declaring our `NSInteger` variables earlier, declaring our `welcome` string requires a `*` ("star") between the class name and the variable name. This is Objective-C's syntax for creating an object.
-
 **Advanced:** *The use of a* `*` *generates a pointer reference. Instead of pointing to a memory location which holds a value, a pointer reference is a location in memory which holds the memory address of another variable's value, or the addresses of a set of other variables' values. [Further Reading](http://www.drdobbs.com/mobile/pointers-in-objective-c/225700236)*
 
-We'll cover in detail the distinction between objects and primitives in a later topic on Object-Oriented-Programming. For now, just recognize that objects hold values and can change them through behaviors called "methods", while primitives just hold a value.
-
-This is because `NSString` is an object type (we'll cover in depth what this means in the next topic). Objects require a pointer reference (the `*`) in their declaration because, well, they point to a set of values at a specific memory address in the computer's RAM. However, `NSInteger` is a primitive, or "data type", and is itself a value, rather than a pointer to a value or set of values.
+We'll cover in detail the distinction between objects and primitives in a later topic on Object-Oriented-Programming. For now, just recognize that objects hold values and can change them through behaviors called "methods", while primitives just hold a value. Because of this, primitives **do not** employ the `*` in the their declaration syntax.
 
 **Note:** *Entering* `NSInteger *` *will cause the compiler to generate an* `invalid integer to pointer conversion` *warning. The same goes for declaring any of the data types. This syntax does serve a purpose so the language permits it, but your application won't do what you expect.*
```

Comment:
Hmm that first sentence is a little misleading. "Assigning an integer to a variable of type `NSInteger *` will cause the compiler..."